josm-presets/de-signals-eso.xml

```diff
@@ -2218,7 +2218,7 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 						de.display_values="Mastbauform,Zwergsignal"
 						/>
 				</item>
-				<item name="Stop/Proceed for Returning Helper Engines (Ts 2/3)" de.name="Halt/Weiterfahrt für zurückkehrende Schieblokomotiven und Sperrfahrten (Ts 2/3)" type="node">
+                <item name="Stop/Proceed for Returning Helper Engines (Ts 2/3)" de.name="Halt/Weiterfahrt für zurückkehrende Schieblokomotiven und Sperrfahrten (Ts 2/3)" icon="de/ts3-32.png" type="node">
```

Comment:
Indentation broken.

Reply:
Sorry, now my setup is improved so that I should see it beforehand...